docs/releases/v0.11.0.rst

```diff
@@ -253,3 +255,4 @@
     1   @mmalohlava
     1   @Viktor-Demin
     1   @chrinide            <Chrinide>
+    1   @pradkrish           <Pradeep Krishnamurthy>
```

Comment:
No need to worry about this, we have a tool that will automatically extract all the contribution details from github. Once released, this information will be included in this changelog.